net/net_evbuffer.c

```diff
@@ -995,16 +995,28 @@ static void check_wr_full(struct event_info *e)
     } else if (outstanding > max_bytes) {
         e->wr_full = time(NULL);
         hprintf("SUSPENDING WR outstanding:%zumb (max:%"PRIu64"mb)\n", outstanding / MB(1), max_bytes / MB(1));
+    } else if (eagain) {
```

Comment:
Good one! Can you collapse this branch into the previous log and instead add `eagain:%d`?

Reply:
Good idea. It's done.